src/common/harvest/harvest.test.js

```diff
@@ -288,6 +289,20 @@ describe('_send', () => {
     })
   })
 
+  test('should warn (once) if payload is large', () => {
+    spec.payload.body = 'x'.repeat(1024 * 1024) // ~1mb string
+
+    const result = harvestInstance._send(spec)
+
+    expect(result).toEqual(true)
+    expect(warn).toHaveBeenCalledWith('The Browser Agent is attempting to send a very large payload. This is usually tied to large amounts of custom attributes. Please check your configurations.')
```

Comment:
String subject to change. `stringContaining` for just the first sentence (`Browser Agent is attempting to send a very large payload`) is more flexible.